src/libcharon/plugins/kernel_super/system_events/empty_system_events/empty_system_events.c

```diff
@@ -0,0 +1,102 @@
+
+#include "empty_system_events.h"
+
+#include "../libsystem_events/libsystem_events.h"
+#include "logger.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include <utils/debug.h>
+
+typedef struct private_empty_system_events_t private_empty_system_events_t;
+
+struct private_empty_system_events_t {
+	empty_system_events_t public;
+};
+
+static void empty_system_events_destroy(system_events_t *_this)
```

Comment:
Again, use `METHOD()` macro for defining method-like functions

Reply:
I thought it is not necessary in here since the function doesn't call anything

Reply:
Doesn't matter, in strongswan all functions that implement an interface's API should be declared with the `METHOD()` macro

Reply:
Ok cool